packages/playwright-test/src/runner/loadUtils.ts

```diff
@@ -147,8 +147,11 @@ export async function createRootSuite(testRun: TestRun, errors: TestError[], sho
   // Complain about only.
   if (config.config.forbidOnly) {
     const onlyTestsAndSuites = rootSuite._getOnlyItems();
-    if (onlyTestsAndSuites.length > 0)
-      errors.push(...createForbidOnlyErrors(onlyTestsAndSuites));
+    if (onlyTestsAndSuites.length > 0) {
+      const forbidOnlyConfigOption = !!config.configCLIOverrides.forbidOnly;
```

Comment:
This is actually `forbidOnlyCLIFlag`.

Reply:
Fixed! I assume `config.configCLIOverrides` is a higher level API over `config.cliArgs`?